src/helpText.jsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
+import { isSafari } from './utils';
 
 /**
  * This file holds a few jsx constants to share helpText between different components.
  * Please keep it sorted for ease of use.
  */
 
+const publishDateHelp = (
+  <div>
+    { isSafari && <p>Required Format: YYYY/MM/DD</p> }
```

Comment:
Nit: I would also have this match the case that we show in the placeholder. So either uppercase in both or lowercase in both.